glauth/pkg/flagset/flagset.go

```diff
@@ -188,12 +195,25 @@ func ServerWithConfig(cfg *config.Config) []cli.Flag {
 			EnvVars:     []string{"GLAUTH_BACKEND_GROUP_FORMAT"},
 			Destination: &cfg.Backend.GroupFormat,
 		},
+		&cli.StringSliceFlag{
+			Name:    "backend-server",
+			Value:   cli.NewStringSlice("https://demo.owncloud.com"),
+			Usage:   `--backend-servers http://internal1.example.com [--backend-servers http://internal2.example.com]`,
```

Comment:
The usage writes "backend-servers" but it is "backend-server"

Reply:
fixed